tests/unit/downloader/test_data_collector.py

```diff
@@ -12,21 +14,25 @@ def setUp(self) -> None:
         self.maxDiff = None
         self.mock_ingest_api = MagicMock(spec=IngestApi)
         self.data_collector = DataCollector(self.mock_ingest_api)
+        self.script_dir = os.path.dirname(__file__)
+        self.parent_dir = os.path.split(self.script_dir)[0]
+        self.resources_dir = os.path.join(self.parent_dir, 'resources')
+
 
     def test_collected_project_and_biomaterials_data_by_submission_uuid_returns_correctly(self):
         #given
         project_uuid = '1234'
-        with open('../resources/mock_submission.json') as file:
+        with open(self.resources_dir + '/mock_submission.json') as file:
```

Comment:
Have you considered extracting a `make_project` function? This would make the test function much more readable.

Reply:
done!